src/JsonApi/Serializer/ErrorNormalizer.php

```diff
@@ -52,6 +52,10 @@ public function normalize($object, $format = null, array $context = [])
             $data['trace'] = $trace;
         }
 
+        if (0 !== $code = $object->getCode()) {
```

Comment:
Could you move this block before the one setting the `trace` key?

Reply:
this is outdated and no longer needed as the ExceptionInterface makes sure getCode returns a string.